contracts/libraries/Account.sol

```diff
@@ -551,7 +572,10 @@ library Account {
         int256 fixFee,
         Protocol.Info storage protocol
```

Comment:
Can remove this function - not being used now

Reply:
It seems to be used in ClearingHouseView. Just to confirm, you mean that external method is not needed over there?

Reply:
I mean "_updateLiquidationAccounts" function